src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -319,6 +319,7 @@ public int RunTest(string executable, string outputFile, string errorFile)
                 process.StartInfo.UseShellExecute = false;
                 process.StartInfo.RedirectStandardOutput = true;
                 process.StartInfo.RedirectStandardError = true;
+                process.StartInfo.EnvironmentVariables.Add("NameSpace", nameSpace);
```

Comment:
this should use a constant similar to `COLLECT_DUMPS_ENVIRONMENT_VAR` instead of hardcoding the name here.

and I think I'd call it `__Namespace` to make it consistent with the other vars. Or maybe it should be `__TestGroup` ?

Reply:
Changed to `__Namespace`